org.civicrm.bootstrapcivihr/scss/overrides/_variables.scss

```diff
@@ -9,13 +9,14 @@ $outsideNamespaceMarker: '.___outside-namespace';
 //== User Menu
 //
 //##
-$user-menu-color: $gray-darker;
-$user-menu-line-height: 60px;
-$user-menu-minify-breakpoint: 1250px;
-$user-menu-bg-hover: $gray-lighter;
-$user-menu-dropdown-bg: $crm-white;
-$user-menu-dropdown-link-color: $gray-darker;
-$user-menu-dropdown-link-bg-hover: $gray-lighter;
+$chr-user-menu-color: $gray-darker;
+$chr-user-menu-line-height: 60px;
+$chr-user-menu-minify-breakpoint: 1250px;
```

Comment:
You don't need to specify this variable since you are already importing the SSP theme and it's defined there.

Reply:
not sure what you mean, i'm not importing the ssp theme here at all